recipes/svgpathtools/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "svgpathtools" %}
+{% set version = "1.4.1" %}
+{% set sha256 = "7aaba07923ad85b6628301e92e5e72fd0d0a5057620e1423509b3a0b609b7485" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+  run:
+    - python >=3.6
+    - numpy
+    - svgwrite
+
+test:
+  # requires:
+    # - scipy
+  source_files:
+    - test
+  commands:
+    # One unit test currently fails under Windows. See also:
+    #     https://github.com/conda-forge/staged-recipes/pull/15429#issuecomment-869240403
+    - python -m unittest discover test  # [not win]
+  imports:
+    - svgpathtools
+
+about:
+  home: https://github.com/mathandy/svgpathtools
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```

I thought I posted this before, but can you try:
```suggestion
  license_file:
  - LICENSE.txt
  - LICENSE2.txt
```

---

**Brilliant!** I wasn't *quite* sure how to effect that and had just discovered conda/conda-build#3600 when your review dropped. I'll do exactly as you say straight-away. :+1:

---

**Ugh.** I committed your awesome suggestion with f439f9f852ae, but GitHub doesn't seem to have picked up on that. <sup>*tnx alot, microsoft*</sup>